doc/manual/running-external-programs.rst

```diff
@@ -314,7 +314,7 @@ When reading and writing to both ends of a pipeline from a single process,
 it is important to avoid forcing the kernel to buffer all of the data.
 
 For example, when reading all of the output from a command,
-call ``readall(out)``, not ``wait(process)``, since the former
+call ``read(out)``, not ``wait(process)``, since the former
```

Comment:
"reading all of the output from a command" probably means as a string more often than as bytes